server/src/main/java/io/spine/server/delivery/CatchUpColumn.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020, TeamDev. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.server.delivery;
+
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.Timestamp;
+import io.spine.query.RecordColumn;
+
+/**
+ * The columns stored for {@link CatchUp} states.
+ */
+@SuppressWarnings("DuplicateStringLiteralInspection") // column names may repeat in different types.
+public class CatchUpColumn {
+
+    /**
+     * Stores the status of the catch-up process.
+     */
+    public static final RecordColumn<CatchUp, CatchUpStatus> status =
+            new RecordColumn<>("status", CatchUpStatus.class, CatchUp::getStatus);
+
+    /**
+     * Stores the time when the history has been last read by the catch-up process.
+     */
+    public static final RecordColumn<CatchUp, Timestamp> when_last_read =
```

Comment:
We probably need to have an annotation for column fields in Java classes to explain why they are named following Protobuf, rather than Java, conventions.